camel/societies/workforce/workforce_logger.py

```diff
@@ -50,6 +60,14 @@ def __init__(self, workforce_id: str):
         self._worker_information: Dict[str, Dict[str, Any]] = {}
         self._initial_worker_logs: List[Dict[str, Any]] = []
 
+    def log_message(self, event: LogEvent) -> None:
+        r"""Just basic print to the console with color."""
+        if event.level == 'info' or event.level not in _COLOR_MAP:
+            print(event.message)
+        color = _COLOR_MAP.get(event.level)
+        colored_message = f"{color}{event.message}{Fore.RESET}"
+        print(colored_message)
```

Comment:
if we use the logger.info/warning/error. module here, the info and success levels does not log to the console out which changes the original behavior of the print statements. do we want to use logger instead?